go/ql/integration-tests/test-extraction-traced/test.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+def test(codeql, go):
+    codeql.database.create(source_root="src", command="go test -c")
```

Comment:
I think the `test-extraction-traced` integration test has the same Go code as `test-extraction-autobuild` and `traced-extraction`? If so, all three could share the same directory with multiple test functions in `test.py` (as per `@redsun82`'s recent talk about this). That would be nice so we don't have duplicate test projects that all need to be updated if we change something.

Reply:
👍 I'll do this as a followup since I'll need to learn how to do this